src/components/NcAppNavigationCaption/NcAppNavigationCaption.vue

```diff
@@ -145,6 +144,8 @@ export default {
 		opacity: $opacity_normal;
 		box-shadow: none !important;
 		flex-shrink: 0;
+		// padding to align the name with the icon of app navigation items
+		padding: 0 calc(var(--default-grid-baseline, 4px) * 2) 0 calc(var(--default-grid-baseline, 4px) * 3);
```

Comment:
I don't understand the syntax, why not just this? 

```suggestion
		padding: 0 calc(var(--default-grid-baseline) * 2) 0 calc(var(--default-grid-baseline) * 3);
```

Reply:
That's the fallback in case `--default-grid-baseline` is not defined.